test/fixtures/elm/queries/SimpleQueries.cql

```diff
@@ -1,35 +0,0 @@
-library SimpleQueries version '0.0.1'
```

Comment:
was this an intentional nuking of this file?

Reply:
nope. Brought it back.